ansible/roles/php70/tasks/main.yml

```diff
@@ -19,6 +19,17 @@
   notify:
     - restart php7.0-fpm
 
+- name: install zip, soap, simple_xml, mbstring
```

Comment:
@dominikzogg I added the task in the config. i don't need the task for php5 because the extensions are installed as default.